### PR TITLE
Move `.vscodeignore` and remove unused `test` script from `package.json`

### PR DESCRIPTION
**Pull request by anuvratsingh**
Fixes Issue #90 

- Moves `.vscodeignore` from root to `vscode-ext`.
- Removes unused script `test` from `vscode-ext/package.json`.

### CODE REVIEW

File: vscode-ext/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"displayName": "mirrord",
 	"description": "Mirror live traffic from your Kubernetes cluster to your local debugged process.",
 	"publisher": "MetalBear",
-	"version": "2.4.1",
+	"version": "1.0.0",
```
Comment (eyalb181) [inline vscode-ext/package.json:6]: Why did you change the version?

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: vscode-ext/package.json (hunk 1 above)
Comment (anuvratsingh) [reply vscode-ext/package.json:6]: That happened because I was playing with `vsce package`, and somehow I missed it while looking at changes.

File: vscode-ext/package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
 	"name": "mirrord",
```
Comment (eyalb181) [inline vscode-ext/package-lock.json:2]: Note that you've left the version as 1.0.0 here

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: vscode-ext/package-lock.json (hunk 2 above)
Comment (anuvratsingh) [reply vscode-ext/package-lock.json]: Sorry, for wasting your time, I messed up such a simple issue.

Comment (eyalb181) [reply vscode-ext/package-lock.json]: No worries at all! Took me like two seconds and we're glad to finally get this issue done :)
